src/pages/en/guides/markdown-content.md

```diff
@@ -406,76 +375,104 @@ const { Content } = Astro.props.post
 </article>
 ```
 
-## Configuring Markdown
+
+### MDX-only Exports
+
+MDX files can also export data with an `export` statement.
+
+For example, you can export a `title` field from an MDX page or component.
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'My first MDX post'
+```
+
+This `title` will be accessible from `import` and [Astro.glob()](/en/reference/api-reference/#astroglob) statements:
+
+```astro
+---
+// src/pages/index.astro
+const posts = await Astro.glob('./*.mdx');
+---
+
+{posts.map(post => <p>{post.title}</p>)}
+```
+
+### Custom components with imported MDX
+
+When rendering imported MDX content, [custom components](#assigning-custom-components-to-html-elements-in-mdx) can be passed via the `components` prop.
+
+Note: Custom components defined and exported in an MDX file must be imported and then passed back to the `<Content />` component via the `components` property.
```

Comment:
I guess this could be a proper `:::note` and maybe makes sense _after_ the code sample?